input/pagecontent/usage.md

```diff
@@ -24,11 +24,6 @@ States and jurisdictions are consistently represented using two letter postal co
 ### Country Literals
 Countries are consistently represented using two letter postal codes bound to a string field.   The IJE typically represents countries using a 2 letter postal code and a separate literal field.  Mapping from the 2 letter codes to literals must be provided outside of the VRDR IG, and can leverage the [CountryCS].
 
-### Decedent Name
-Profile conformance requires that either a first, last, or middle name be provided based on US Core requirements.  If no name components are available, a data absent reason should be provided for at least one name component in order to conform to the profile requirements.
-
-NCHS submission requires only a last name. Last name is required for NCHS submission.  If the decedent's legal name is unknown, name.family should be reported as 'UNKNOWN', or omitted. Placeholder names such as 'John Doe' or 'Jane Doe' should not be reported.
```

Comment:
Sorry.... For VRDR we still need the note on NCHS submission.  I added it after there was confusion based on the historic practice and the lack of documentation in  the IG.

Reply:
Is this for VRDR or VRCL? I was removing it from VRCL, because I think it would make more sense for this note to live in VRDR, as it is referring to the Decedent profile which lives in VRDR

Reply:
Perfect.   Sorry for the misunderstanding.   Noone should be looking in VRCL for use-case specific directions.   We need to clarify that somewhere.